autoload/codefmt.vim

```diff
@@ -98,18 +95,20 @@ function! codefmt#EnsureFormatter(formatter) abort
 endfunction
 
 
+""
+" @private
 " Formatter: js-beautify
-if !exists('s:js_beautify')
-  let s:js_beautify = {
+function! codefmt#GetJsBeautifyFormatter() abort
+  let l:formatter = {
       \ 'name': 'js-beautify',
       \ 'setup_instructions': 'Install js-beautify ' .
           \ '(https://www.npmjs.com/package/js-beautify).'}
 
-  function s:js_beautify.IsAvailable() abort
+  function l:formatter.IsAvailable() abort
```

Comment:
Wow, does this actually work? I thought trying to nest function definitions in vim did something weird like ending both the inner and outer function at the first `:endfunction`.

If it does work properly I'm fine with it. Otherwise you can just add `#GetJsBeautifyFormatter` as a small wrapper that returns `s:js_beautify`.